semgrep-core/tests/go/package2.sgrep

```diff
@@ -0,0 +1 @@
+package $X
```

Comment:
wrong PR, but yes you can call those tests misc_package2.sgrep. We try to follow some conventions in the test filenames (which are then used to generate the cheatsheets), so for those features, just put a misc_prefix.